src/js/jsx/sections/export/ExportList.jsx

```diff
@@ -160,6 +163,8 @@ define(function (require, exports, module) {
                 keySuffix = this.props.faceKey,
                 scaleListID = "exportAsset-scale-" + keySuffix,
                 formatListID = "exportAsset-format-" + keySuffix;
+            
+            headlights.logEvent("export", "create-asset", formatOptionId);
```

Comment:
I think it's a bad idea to put this call in the render method, because the render method may be called many times for the same asset. Can you put it in the click handler that adds the asset?